static/js/lib/base64.js

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Definitely worth a comment, I meant to add one to the file but I forgot. Will do.